modules/lemmaBidAdapter.js

```diff
@@ -62,18 +62,6 @@ export var spec = {
   },
   getUserSyncs: (syncOptions, responses, gdprConsent, uspConsent) => {
     let syncurl = USER_SYNC + 'pid=' + pubId;
-
-    // Attaching GDPR Consent Params in UserSync url
-    if (gdprConsent) {
-      syncurl += '&gdpr=' + (gdprConsent.gdprApplies ? 1 : 0);
-      syncurl += '&gdpr_consent=' + encodeURIComponent(gdprConsent.consentString || '');
-    }
-
-    // CCPA
-    if (uspConsent) {
-      syncurl += '&us_privacy=' + encodeURIComponent(uspConsent);
-    }
-
```

Comment:
Why removing gdpr and ccpa support?  
If you really need to remove this, please do it in separate PR. This PR should be only about floor prices.

Reply:
Hi, 
Above changes already merged into master 
https://github.com/prebid/Prebid.js/pull/6006